components/Feature/TopicExplorer/index.test.js

```diff
@@ -34,16 +34,24 @@ describe('TopicExplorer', () => {
       expect(screen.queryByText('topic two')).toBeNull();
     });
 
+    it('shows autocomletion of search terms', () =>{
```

Comment:
nit: typo (I've tweaked the phrasing slightly too :speak_no_evil:)
```suggestion
    it('shows auto-complete for search terms', () =>{
```

Reply:
I've updated this as part of the previous commit 😄